ocaml/xapi-types/features.ml

```diff
@@ -165,4 +165,8 @@ let of_assoc_list l =
     with _ ->
       if List.mem f enabled_when_unknown then Some f else None
   in
-  Stdext.Listext.List.filter_map get_feature all_features
+  (* Filter_map to avoid having to carry the whole xapi-stdext-std *)
+  List.fold_right (fun f acc ->
```

Comment:
Not that it matters in this case but this is not tail recursive.

Reply:
True, I thought that `List.fold_left` and then `rev`, or a custom rewrite, was overkill for the feature list. I'll add a warning to the comment for future reference

Reply:
Sounds good, just to prevent someone accidentally using it as an example for writing new code.